docs/building.rst

```diff
@@ -647,3 +655,22 @@ in a command shell::
 
 This will also install the headers and library files in the visual studio directories to
 be picked up by libtorrent.
+
+list of macros
+--------------------
```

Comment:
you use too many `-` characters here. To make it a header you need as many `-` as there are characters in the heading itself

Reply:
Got it! Definitely need to ramp my knowledge of RST format.